src/types/types.ts

```diff
@@ -0,0 +1,14 @@
+export type User = {
```

Comment:
```suggestion
export interface User = {
```
use interface prefer